eng/tools/generator/cmd/v2/common/testdata/new/func/funcfilter.go

```diff
@@ -0,0 +1,12 @@
+package funcfilter_test
```

Comment:
This should be `package func` (func is a keyword, maybe you need to rename this directory)